assets/src/paypal/paymenthelper.js

```diff
@@ -242,7 +242,7 @@ module.exports = {
 			if (context.configuration && context.configuration.paypal && context.configuration.paypal.capture)
 			paymentAction.amount = context.configuration.paypal.capture.amount;
 
-			return client.doCapture(payment.externalTransactionId,order.orderNumber,
+    return client.doCapture(payment.externalTransactionId, paymentAuthorizationInteraction.target.targetNumber,
```

Comment:
Use this field from interaction only if present. If not present, fall back to order number.
Also add code comment-
"Using interaction target field to prefer sending checkout number for multiship order. In case of multiship, authorization always happens at checkout level"

Reply:
Done